### PR TITLE
List available carriers and make offer

### PR DESCRIPTION
**Pull request by tommysr**

### CODE REVIEW

**Comment by tommysr** [comment]
@matzayonc ready to deploy if approved and merged

File: app/src/routes/shipments/bought/+page.ts
Comment (tejks) [inline app/src/routes/shipments/bought/+page.ts:1]: It should be on the server side

File: app/src/sass/main.scss
Comment (tejks) [inline app/src/sass/main.scss:1]: Changed in branch: https://github.com/tommysr/trans-protocol/tree/styles-rework

File: app/src/components/Buttons/SimpleButton.svelte
Comment (tejks) [inline app/src/components/Buttons/SimpleButton.svelte:8]: If the classes are dynamic, this might make sense, but static classes don't need to be put in a separate variable

Comment (tejks) [inline app/src/components/Buttons/SimpleButton.svelte:8]: If the classes are dynamic, this might make sense, but static classes don't need to be put in a separate variable

File: app/src/routes/shipments/bought/+page.ts
Comment (tejks) [inline app/src/routes/shipments/bought/+page.ts:1]: Should be fetched on the server side

File: app/src/components/Buttons/SimpleButton.svelte
Comment (tommysr) [reply app/src/components/Buttons/SimpleButton.svelte:8]: I did it to concatenate class names and it does not break anything.

File: app/src/routes/shipments/bought/+page.ts
Comment (tommysr) [reply app/src/routes/shipments/bought/+page.ts:1]: I think it can't be, because page has a map and map needs widow object

File: app/src/components/Buttons/SimpleButton.svelte
Comment (tommysr) [reply app/src/components/Buttons/SimpleButton.svelte:8]: should i do it some other way?

File: app/src/components/Offer/OfferDetailsModal.svelte
Comment (tommysr) [inline app/src/components/Offer/OfferDetailsModal.svelte:4]: should be the other way around, stupid copilot

**Review by matzayonc — APPROVED** [review]

File: app/src/components/Shipment/OrderCard.svelte
Comment (matzayonc) [inline app/src/components/Shipment/OrderCard.svelte:76]: maybe just connect it instead

File: app/src/routes/shipments/bought/+page.svelte
Comment (matzayonc) [inline app/src/routes/shipments/bought/+page.svelte:113]: same

**Review by matzayonc — APPROVED** [review]

File: app/src/components/Offer/OfferDetailsModal.svelte
Comment (tommysr) [reply app/src/components/Offer/OfferDetailsModal.svelte]: fixed

File: app/src/components/Shipment/OrderCard.svelte
Comment (tommysr) [reply app/src/components/Shipment/OrderCard.svelte]: resolved, please review @matzayonc

File: app/src/routes/shipments/bought/+page.svelte
Comment (tommysr) [reply app/src/routes/shipments/bought/+page.svelte]: same, resolved

File: app/src/components/Buttons/SimpleButton.svelte
Comment (tommysr) [reply app/src/components/Buttons/SimpleButton.svelte:8]: @tejks bump

Comment (tejks) [reply app/src/components/Buttons/SimpleButton.svelte:8]: You can use clsx library or or insert variable with ${}.

**Comment by tommysr** [comment]
This have to wait until #31 is merged

File: app/src/stores/walletModalHelper.ts
Comment (tejks) [inline app/src/stores/walletModalHelper.ts:1]: Is a separate modal store needed?

Comment (tommysr) [reply app/src/stores/walletModalHelper.ts:1]: fixed

**Comment by tommysr** [comment]
> ### This have to wait until #31 is merged

don't have to #33 is here 👯‍♂️

**Pull request merged**
